Pensel/firmware/modules/utilities/scheduler.c

```diff
@@ -0,0 +1,118 @@
+/*!
+ * @file    scheduler.c
+ * @author  Tyler Holmes
+ * @version 0.1.0
+ * @date    28-April-2018
+ * @brief   A rudamentary scheduler
+ */
+#include <stdint.h>
+#include <stdbool.h>
+
+#include "common.h"
+#include "scheduler.h"
+
+// TODO remove - debug
+// #include "peripherals/UART/UART.h"
+
+
+ret_t scheduler_init(schedule_t * schedule)
+{
+    schedule->num_slots_used = 0;
+    for (uint8_t i=0; i < NUM_SLOTS; i++) {
```

Comment:
nit: spaces around `=`

Reply:
I’ve gone back and forth on this style. I think I might like what is there from python coding